app/src/main/res/values/strings.xml

```diff
@@ -776,4 +776,13 @@
   <string name="Announce.no_thanks">No, thanks!</string>
     <string name="Fees.Service">Service</string>
     <string name="ConfirmationAllFees.Label">Fees:</string>
+
+  <string name="notification_channel_name_general" translatable="false">General</string>
```

Comment:
currently I just create this new copy, do we have to translate into all of languages?

Reply:
I'm not sure. What do you think @kcw-grunt ?

Reply:
for channel name maybe no need to translate, but like `please_grant_notification_permission` I think we need to translate